lib/PerlIO.pm

```diff
@@ -366,8 +477,7 @@ You are supposed to use open() and binmode() to manipulate the stack.
 B<Implementation details follow, please close your eyes.>
 
 The arguments to layers are by default returned in parentheses after
-the name of the layer, and certain layers (like C<:utf8>) are not real
```

Comment:
This also is still the case, even if not for :utf8.

Reply:
I've re-worked this, only the F_UTF8 flag ended up as a layer, and no longer does.